third_party/libvpx/BUILD.gn

```diff
@@ -7,7 +7,13 @@ import("//build/config/arm.gni")
 import("//build/config/chromeos/ui_mode.gni")
 import("//build/config/sanitizers/sanitizers.gni")
 import("//third_party/libvpx/libvpx_srcs.gni")
-import("//third_party/nasm/nasm_assemble.gni")
+if (is_starboard) {
```

Comment:
Does libvpx get built with the native toolchain as well?

Reply:
I don't actually know, I included these lines since it seemed to be the convention when including nasm assemble.

Reply:
Done